src/core/AutoRest/AutoRest.json

```diff
@@ -35,6 +35,9 @@
     },
     "AzureResourceSchema": {
       "type": "AzureResourceSchemaCodeGenerator, AutoRest.AzureResourceSchema"
+    },
+    "Go": {
```

Comment:
@vishrutshah Sorry I was wrong earlier. I discussed with Brendan, this is a generic generator.